src/systemtest/java/com/ericsson/ei/systemtest/utils/JenkinsManager.java

```diff
@@ -0,0 +1,215 @@
+package com.ericsson.ei.systemtest.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import javax.xml.bind.DatatypeConverter;
+
+import org.apache.http.client.ClientProtocolException;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+
+import com.ericsson.ei.utils.HttpRequest;
+import com.ericsson.ei.utils.HttpRequest.HttpMethod;
+
+public class JenkinsManager {
+
+    private static final String JENKINS_JOB_TEMPLATE_FILE_PATH = String.join(File.separator, "src", "systemtest",
+            "resources", "jenkinsJobTemplate.xml");
+    private static final String DEFAULT_JOB_TOKEN = "test";
+    private static final String DEFAULT_SCRIPT = "echo &quot;Test&quot;";
+    private static final String DEFAULT_JOB_NAME = "test_job";
+
+    private String host;
+    private int port;
+    private String encoding;
+    private String crumb;
+
+    /**
+     * Constructor, takes jenkins host, port, username and password.
+     * @param host
+     * @param port
+     * @param username
+     * @param password   **  Jenkins password or API-token
+     * @throws UnsupportedEncodingException
+     * @throws URISyntaxException
+     * @throws JSONException
+     */
+    public JenkinsManager(String host, int port, String username, String password)
```

Comment:
HTTPS need to handled here as well. Maybe easier if we have "jenkins.url" property in application properties,, then we maybe not need to care about the Https or http handling, since its inclided in the jenkins.url property.